subworkflows/local/tfea/homer.nf

```diff
@@ -0,0 +1,36 @@
+/*
+ * TFEA by homer
+ */
+
+include { HOMER_FINDMOTIFSGENOME     } from '../../../modules/local/homer/find_motifs_genome'
+include { SEQLEVELS_STYLE            } from '../../../modules/local/bioc/seqlevelsstyle'
+include { ENSEMBL_UCSC_CONVERT       } from '../../../modules/local/bioc/ensembl_ucsc_convert'
+
+workflow HOMER_TFEA {
+    take:
+    bed                      // peaks regions [meta, R1/2, [peak]]
+    additional_param         // singals for each tools
+
+    main:
+    ch_versions = Channel.empty()
+
+    if(!workflow.containerEngine){
```

Comment:
So I'm guessing, not working with conda

Reply:
I need to double check about this. The homer results will be different if there is no pre-installed genome files (the pipeline will try to install the files). For containers, the pipeline will use custom genomes and annotation files 'on-the-fly' because the homer folder are not writable.